shell/platform/android/android_environment_gl.cc

```diff
@@ -25,7 +25,7 @@ AndroidEnvironmentGL::AndroidEnvironmentGL()
 
 AndroidEnvironmentGL::~AndroidEnvironmentGL() {
   // Diconnect the display if valid.
```

Comment:
Unrelated but noticed the typo in the comment above.

Reply:
done